src/Nethermind/Nethermind.Runner/configs/chiado.cfg

```diff
@@ -0,0 +1,46 @@
+{
+  "Init": {
+    "MemoryHint": 768000000,
+    "ChainSpecPath": "chainspec/chiado.json"
+  },
+  "Sync": {
+    "FastSync": true,
+    "FastBlocks": true,
+    "UseGethLimitsInFastBlocks": false,
+    "FastSyncCatchUpHeightDelta": 10000000000
+  },
+  "JsonRpc": {
+    "Enabled": true,
+    "Timeout": 20000,
+    "Host": "127.0.0.1",
+    "Port": 8545,
+    "AdditionalRpcUrls": [
+      "http://localhost:8551|http;ws|net;eth;subscribe;engine;web3;client"
+    ]
+  },
+  "AuRaMerge": {
+    "Enabled": true
+  },
+  "Aura": {
+    "TxPriorityContractAddress": "0x4100000000000000000000000000000000000000",
+    "ForceSealing": true,
+    "AllowAuRaPrivateChains": true
+  },
+  "Merge": {
+    "Enabled": true,
+    "TerminalTotalDifficulty": 115792089237316195423570985008687907853269984665640564039457584007913129639935
```

Comment:
Warning! TTD for chiado is already scheduled `70560951604725799783765358597051456327101440` from https://github.com/gnosischain/consensus-deployment-ansible/tree/master/chiado#chiado-testnet

Reply:
This TTD value is set twice, here and in the genesis file. Should you set it only in one place? Now it's inconsistent